tools/cldr-apps/js/src/views/DashboardPanel.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div id="dashboard">
+    <p v-if="fetchErr" class="st-sad">Error loading data: {{ fetchErr }}</p>
+    <p v-if="!data" class="st-sad">Loading…</p>
+    <div v-if="data && !fetchErr" id="scrollingwindow">
+        <div v-for="n in data.notifications" :key="n.notification">
+            <h3 class="collapse-review">
+                {{ n.notification }}
+            </h3>
+            <div v-for="g in n.entries" :key="g.header">
+                <h4>{{ g.section }}/{{ g.page }}/{{ g.header }}</h4>
+                <div v-for="e in g.entries" :key="e.xpath">
+                        <a v-bind:href="'#/'+[locale, g.page, e.xpath].join('/')">
+                    <span
+                        class="label label-info">
+                            {{ e.code }}
+                    </span> </a><br/>
+                    English: {{ e.english }}<br />
+                    Old: {{ e.old }}<br />
+                    Winning: {{ e.winning }}<br />
+                    <i span v-html="e.comment"></i><br />
+                    <hr />
+                </div>
+            </div>
+        </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  props: ["specialPage", "cldrOpts"],
+  data: function () {
+    return {
```

Comment:
alternative syntax: `data()` instead of `data: function()`, maybe worth considering

Reply:
:+1: thanks, i'll use that.

Reply:
didn't know that would work… 